apps/ae_cuckoo_cycle_pow_executables/test/ae_cc_pow_executables_SUITE.erl

```diff
@@ -0,0 +1,47 @@
+%%%-------------------------------------------------------------------
+%%% @copyright (C) 2017, Aeternity Anstalt
+%%% @doc Basic sanity checks and examples on Cuckoo cycle PoW executables.
+%%% @end
+%%%-------------------------------------------------------------------
+-module(ae_cc_pow_executables_SUITE).
+
+%% common_test exports
+-export([all/0]).
+
+%% test case exports
+-export([smoke_test/1]).
+
+-include_lib("common_test/include/ct.hrl").
+
+-define(TEST_MODULE, ae_cuckoo_cycle_pow_executables).
+
+all() ->
+    [smoke_test].
+
+smoke_test(_Config) ->
+    LibDir = ?TEST_MODULE:lib_dir(),
+    MinBin = ?TEST_MODULE:bin("lean16"),
+    VerBin = ?TEST_MODULE:bin("verify16"),
+    LdLibPathVarName = ld_lib_path_var_name(),
+    Cmd =
+        io_lib:format(
+          "env ~s='~s' '~s' -n 66 | grep '^Solution ' | env ~s='~s' '~s' -n 66 | grep '^Verified '",
```

Comment:
I would prefer using the ability to break the string across multiple lines, e.g.:

```erlang
io_lib:format("env ~s='~s' '~s' -n 66 "
              "| grep '^Solution '"
              "| env ~s='~s' '~s' -n 66 "
              "| grep '^Verified '",
              [...])
```

Reply:
Addressed in next commit 6c57766